.github/workflows/applications.yml

```diff
@@ -355,7 +355,7 @@ jobs:
         ghc: ${{ matrix.ghc }}
     - name: Run all benchmarks
       timeout-minutes: 12
-      run: bench --json $BENCH_FILE +RTS -N
+      run: bench --json $BENCH_FILE +RTS -H4G -K512m -RTS
```

Comment:
I think, we should not remove it. In production we run with `-N`, and even though this is "just" a benchmark it would reduce coverage or my skew the results. It should not fail with `-N`, so removing it, potentially hide a bug. 

We could, however, consider hardcoding it to a reasonably small value, say `-N4` or `-N2`.

Reply:
@edmundnoble what are your thoughts on this? What about using `-N2` or `-N4`?

Reply:
While i don't think we want to run these benchmarks in serial, github runners have 2 cores dedicated for linux/windows and 3 for macos. Seems like `-N4` would make the most sense to me.